demos/betting-behavior-analysis.mdx

```diff
@@ -0,0 +1,212 @@
+
+During live events, users’ behaviors can shift rapidly in response to gameplay developments.
+Processing and analyzing these changes in real-time allows platforms to flag high-risk users, who may be more likely to engage in fraudulent activities.
+By joining historic data on user behavior with live betting data, platforms can easily identify high-risk users for further investigation to mitigate potential risks.
+
+In this tutorial, you will learn how to analyze users’ betting behaviors by integrating historical datasets with live data streams.
+
+## Prerequisites
+
+* Ensure that the [PostgreSQL](https://www.postgresql.org/docs/current/app-psql.html) interactive terminal, `psql`, is installed in your environment. For detailed instructions, see [Download PostgreSQL](https://www.postgresql.org/download/).
+* Install and run RisingWave. For detailed instructions on how to quickly get started, see the [Quick start](/get-started/quickstart.mdx) guide.
+* Ensure that a Python environment is set up and install the [psycopg2](https://pypi.org/project/psycopg2/) library. 
+
+## Step 1: Set up the data source tables
+
+Once RisingWave is installed and deployed, run the three SQL queries below to set up the tables. You will insert data into these tables to simulate live data streams.
+
+1. The table `user_profiles` table contains static information about each user.
+
+    ```sql
+    CREATE TABLE user_profiles (
+        user_id INT,
+        username VARCHAR,
+        preferred_league VARCHAR,
+        avg_bet_size FLOAT,
+        risk_tolerance VARCHAR
+    );
+    ```
+
+2. The `betting_history` table contains historical betting records for each user.
+
+    ```sql
+    CREATE TABLE betting_history (
+        user_id INT,
+        position_id INT,
+        bet_amount FLOAT,
+        result VARCHAR,
+        profit_loss FLOAT, 
+        timestamp TIMESTAMP
+    );
+    ```
+
+3. The `positions` has real-time updates for ongoing betting positions for each user.
+
+    ```sql
+    CREATE TABLE positions (
+        position_id INT,
+        position_name VARCHAR,
+        user_id INT,
+        league VARCHAR,
+        stake_amount FLOAT,
+        expected_return FLOAT,
+        current_odds FLOAT,
+        profit_loss FLOAT,
+        timestamp TIMESTAMP
+    );
+    ```
+
+## Step 2: Run the data generator
+
+To keep this demo simple, a Python script is used to generate and insert data into the tables created above. 
+
+Navigate to the [user_betting_behavior](https://github.com/risingwavelabs/awesome-stream-processing) folder in the [awesome-stream-processing](https://github.com/risingwavelabs/awesome-stream-processing) and run the `data_generator.py` file. This Python script utilizes the `psycopg2` library to establish a connection with RisingWave so you can generate and insert synthetic data into the tables `user_profiles`, `betting_history`, and `positions`. 
```

Comment:
Can we directly link to the folder instead of repository root `https://github.com/risingwavelabs/awesome-stream-processing`?

Reply:
The folders haven't been merged to the repository yet so the root folder is a place holder for now